pkg/subctl/operator/deploy/ensure.go

```diff
@@ -0,0 +1,75 @@
+package deploy
+
+import (
+	"fmt"
+
+	submariner "github.com/submariner-io/submariner-operator/pkg/apis/submariner/v1alpha1"
+	submarinerclientset "github.com/submariner-io/submariner-operator/pkg/client/clientset/versioned"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/datafile"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/rest"
+)
+
+func Ensure(config *rest.Config, remoteNamespace string, submarinerNamespace string,
+	clusterID string, serviceCIDR string, clusterCIDR string,
+	subctlData *datafile.SubctlData) error {
+
+	fmt.Printf("%v\n", string(subctlData.ClientToken.Data["ca.crt"]))
+
+	// Populate the Submariner CR for the operator
+	submarinerSpec := submariner.SubmarinerSpec{
```

Comment:
Yay! This will free up lots of Bash for removal.

Reply:
Wow yeah!